### ISSUE TITLE
Export test to JS code for offline usage

### ISSUE THREAD

**Issue by scspijker** [psiho/jsbench-me#59]
I love JSBench 👍 

Our backend uses Node, and I would really like to be able to export JSBench test to standard Javascript to run this in our own Javascript environment instead of in a browser.

An example would be this test: https://jsbench.me/o8kqzo8olz/1

**Comment by psiho** [comment]
Good idea. But I'm still not putting it to v1.2 milestone because I don't want to postpone that release for any longer. But let's keep this open and see if I can squeeze that in or not, once I resolve higher priority issues and features.
